.env.example

```diff
@@ -1,5 +1,6 @@
 CTF_SPACE_ID="ABC123"
 CTF_CDA_ACCESS_TOKEN="ABCDEF123456"
+CTF_PREVIEW_CDA_ACCESS_TOKEN="GHIJKLM7891011"
```

Comment:
Contentful abbreviate the Preview API as CPA. Let's therefore name this `CTF_CPA_ACCESS_TOKEN`.